carbon-design-system/carbon
# fix(filename): provide aria-describedby to errored filenames

Closes #11202 

#### Changelog


**Changed**

- provide `aria-describedby` to errored filenames, associating the existing error message with the button

#### Testing / Reviewing

- Open the fileuploader `drag-and-drop-upload-container-example-application` story
- Upload a file over 500kb to get the error message
- JAWS should read the error message when it pops (Andrea and I confirmed this with JAWS)
